test/test_baseline_trainers.py

```diff
@@ -140,10 +134,56 @@ def test_cubemap_stiching(test_cfg_path: str, mode: str, camera: str):
             sensor_uuids
         )
     meta_config.freeze()
-    execute_exp(meta_config, mode)
-    # Deinit processes group
-    if torch.distributed.is_initialized():
-        torch.distributed.destroy_process_group()
+    if camera in ["equirect", "fisheye"]:
+        execute_exp(meta_config, mode)
+        # Deinit processes group
+        if torch.distributed.is_initialized():
+            torch.distributed.destroy_process_group()
+
+    elif camera == "cubemap":
+        # 1) Generate an equirect image from cubemap images.
+        # 2) Generate cubemap images from the equirect image.
+        # 3) Compare the input and output cubemap
+        env = habitat.Env(config=config)
```

Comment:
Nit but the Env should be closed. May be good to use a with Env context.

Reply:
Can we actually test a habitat_env wrapped in a vector env? I just want to make sure it doesn't break with a batch size more than 1. :)

Reply:
You can also use the same input and just batch two separate observations (from two separate positions, I am worried about the cubemap values being mixed up potentially and we should test explicitly for that.

Reply:
Thank you. I added `VectorEnv` in testing.